### PR TITLE
647 annot creation without encounter only permitted during curation

### PR DESCRIPTION
**Pull request by hwindsor**
## Pull Request Overview

- Annotation creation without encounter only permittted if AGS is in Curation stage
- Tests updated accordingly

---

### CODE REVIEW

File: app/modules/annotations/resources.py [hunk 1]
```diff
@@ -87,7 +88,26 @@ def post(self, args):
             if not encounter:
                 abort(code=HTTPStatus.BAD_REQUEST, message='encounter_guid not found')
             args['encounter_guid'] = encounter_guid
-
+        elif hasattr(asset.git_store, 'get_asset_group_sightings_for_asset'):
```
Comment (hwindsor) [inline app/modules/annotations/resources.py:91]: @bluemellophone, this is effectively a way of asking if the git_store is an AssetGroup. 
This change has specifically not done anything WRT MWS. 
Currently MWS can create an encounter less annotation at any point. 
I'd suggest that you may want to thinnk about this ...... under a separate PR ;)

File: tests/modules/annotations/resources/test_annotation_keywords.py [hunk 2]
```diff
@@ -29,17 +26,18 @@ def test_keywords_on_annotation(
     with db.session.begin():
         db.session.add(keyword)
     assert keyword is not None
-
```
Comment (hwindsor) [inline tests/modules/annotations/resources/test_annotation_keywords.py:32]: The cloned Asset group has no sightings so is effectively an invaid asset group. So we really need to replace all uses of the cloned AGs with creating ones like below.

**Review by colinwkingen — APPROVED** [review]
LGTM

**Pull request merged**